code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -0,0 +1,389 @@
+	//Baseline hardsuits
```

Comment:
Don't indent comments that aren't inside a path or proc's scope, please.

Reply:
Indented comments have been removed.